tox.ini

```diff
@@ -1,13 +1,23 @@
 [tox]
 skipsdist = true
-envlist = discopane-ui-tests, flake8
+envlist = discopane-ui-tests, flake8, smoke-tests
 
 [testenv:discopane-ui-tests]
 passenv = DISPLAY MOZ_HEADLESS PYTEST_ADDOPTS PYTEST_BASE_URL
 deps = -r{toxinidir}/tests/ui/requirements/requirements.txt
 install_command = pip install --no-deps {packages}
 commands = pytest --driver Firefox tests/ui/tests/test_discopane.py {posargs}
 
+[testenv:smoke-tests]
+basepython = python3.7
+recreate = true
+passenv = DISPLAY MOZ_HEADLESS PYTEST_ADDOPTS
+deps = 
+  pytest-base-url
+  pytest-selenium
+  pytest-rerunfailures
+commands = pytest --driver Firefox --base-url http://localhost:3000 --verify-base-url --reruns 1 tests/smoke/test_homepage.py {posargs}
```

Comment:
Are you sure this host is right? I'd also like to see a README added to the `tests/smoke` directory explaining how to run this locally. I am not sure how to get it working.